lighthouse-core/gather/gatherers/trace-elements.js

```diff
@@ -137,6 +137,26 @@ class TraceElements extends Gatherer {
     return topFive;
   }
 
+  /**
+   * Find the node ids of elements which are animated using the Animation trace events.
+   * @param {Array<LH.TraceEvent>} mainThreadEvents
+   * @return {Array<TraceElementData>}
+   */
+  static getAnimatedElements(mainThreadEvents) {
+    const animatedElementIds = [...new Set(mainThreadEvents
```

Comment:
was the array -> set -> array an artifact of the code before?

might make more sense to avoid the unnecessary array copy and just do `for (const nodeId of idSet)` instead